ruby_on_rails/uptimerobot.md

```diff
@@ -0,0 +1,33 @@
+# Uptimerobot Monitoring
+
+To ensure that our application is always up and running, we offer a monitoring
+service to the customers.
+
+When we are still developing a new application, the uptimerobot check should not be
+setup to avoid premature costs. Once the go-live date is very close, we enable
+the monitoring only for the `master` environment, which *must* have a paid
+dyno.
+
+## Setup
+
+You will need Renuo-CLI to be set up and at the newest version:
+`gem install renuo-cli` --> see [renuo-cli](https://github.com/renuo/renuo-cli)
+
+1. Run the command `renuo setup-uptimerobot [url] [scope]`
+   * Where `url` is the address that you want to setup monitoring for
```

Comment:
`https://[project-name]-master.renuoapp.ch/home/check` or the `https://customdomain/home/check`